extensions/narayana-lra/deployment/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-narayana-lra-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-narayana-lra-deployment</artifactId>
+    <name>Quarkus - Narayana MicroProfile LRA Participant - Deployment</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-narayana-lra</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-jsonb-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-deployment</artifactId>
```

Comment:
Did you try with the reactive rest client (even used in a blocking manner)? It would be better to use the reactive one as it's more integrated with the Quarkus internals. 

CC @michalszynkiewicz

Reply:
I think the client is by [this class](https://github.com/jbosstm/narayana/blob/master/rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java). If it is so, the async api seems to be the only one in use.

Reply:
I will give it a go

Reply:
Yes @michael-schnell we only use the async API. Does that mean I don't need to use the reactive rest client.

NB When I switch to using `<artifactId>quarkus-rest-client-reactive-deployment</artifactId>` I get the following runtime error:
            `ClassNotFoundException: io.quarkus.resteasy.reactive.common.runtime.ResteasyReactiveConfig`
Is is worth debugging this given that we are already using the async version of the API?